File: Plouton-master/models/Category.js
import mongoose from 'mongoose';

const categorySchema = new mongoose.Schema({
  name: {
    type: String,
    required: true,
    unique: true,
    trim: true
  },
  description: {
    type: String,
    required: true
  },
  icon: {
    type: String,
    default: 'default-icon'
  },
  color: {
    type: String,
    default: '#007AFF'
  },
  isActive: {
    type: Boolean,
    default: true
  }
}, {
  timestamps: true
});

const Category = mongoose.model('Category', categorySchema);
export default Category; 

File: Plouton-master/app/src/main/java/com/plouton/CiviLized/data/model/Category.kt
package com.plouton.CiviLized.data.model

import com.google.gson.annotations.SerializedName

data class Category(
    @SerializedName("_id")
    val id: String,
    val name: String,
    val description: String,
    val icon: String,
    val color: String,
    val isActive: Boolean = true,
    val createdAt: String? = null,
    val updatedAt: String? = null
) 